.changeset/hungry-pumas-repair.md

```diff
@@ -0,0 +1,4 @@
+---
+'@swisspost/design-system-documentation': minor
+---
+feat(docs): added package-info to compo on docs.
```

Comment:
Only the **commit messages** start with `feat`, `fix`, or `chore`: these are short messages meant for Design System developers to quickly understand what a commit is about. 

Here you are writing a **change set** meant for external users to understand our changes. These users do not know our code base and need complete sentences with no abbreviations or technical jargon.